packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -0,0 +1,437 @@
+/**
+ * WordPress dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+	TextControl,
+	CheckboxControl,
+} from '@wordpress/components';
+import { dateI18n, format } from '@wordpress/date';
+import {
+	Icon,
+	edit as editIcon,
+	trash as deleteIcon,
+	commentAuthorAvatar as userIcon,
+} from '@wordpress/icons';
+
+/**
+ * Renders the new comment form.
+ *
+ * @param {Object}   root0                   The component props.
+ * @param {Function} root0.setReloadComments Function to reload comments.
+ */
+export function AddComment( { setReloadComments } ) {
+	const generateNewComment = () => ( {
+		commentId: Date.now(),
+		createdBy: currentUser,
+		comment: inputComment,
+		createdAt: new Date().toISOString(),
+	} );
+
+	// Get the date time format from WordPress settings.
+	const dateTimeFormat = 'h:i A';
+
+	// State to manage the comment thread.
+	const [ inputComment, setInputComment ] = useState( '' );
+	const [ isEditing, setIsEditing ] = useState( null );
+
+	const curruntUserData = useSelect( ( select ) => {
+		// eslint-disable-next-line @wordpress/data-no-store-string-literals
+		return select( 'core' ).getCurrentUser();
+	}, [] );
+
+	let userAvatar =
+		'https://secure.gravatar.com/avatar/92929292929292929292929292929292?s=48&d=mm&r=g';
+	if ( curruntUserData?.avatar_urls ) {
+		userAvatar = curruntUserData?.avatar_urls[ 48 ];
```

Comment:
Absolutely, we can discuss that. However, since this is the initial phase, we are currently keeping the size hardcoded to maintain the desired look and feel.